openbas-front/src/admin/components/common/Context.ts

```diff
@@ -60,6 +62,12 @@ export type VariableContextType = {
   onDeleteVariable: (variable: Variable) => void,
 };
 
+export type ReportContextType = {
```

Comment:
If report can be used only on exercise, I am not sure that we need to add a context.
@guillaumejparis what do you think ?